test/collect_coverage_api_test.dart

```diff
@@ -92,6 +101,11 @@ Future<Map<String, dynamic>> _collectCoverage(
     }
   });
   final Uri serviceUri = await serviceUriCompleter.future;
+  Set isolateIdSet;
+  if (isolateIds) {
+    isolateIdSet = Set<String>.from(<String>[]);
+  }
```

Comment:
Merge this assignment the declaration  and make the variable final; also prefer the default constructor for the empty set. I'd have recommended the new set literal syntax but it's not available until Dart 2.2 and coverage is still lower-bounded by SDK 2.1.1.
```
final Set<String> isolateIdSet = isolateIds ? Set() : null;
``

Reply:
Done